src/app/navBar/NavbarComponent.jsx

```diff
@@ -3,6 +3,7 @@ import { Link } from 'react-router-dom';
 import './navBar.scss';
 
 const NavbarComponent = () => {
+  const isSignedIn = localStorage.getItem('token') !== null;
```

Comment:
Is the comparison to null here necessary? We can use the double bang to cast to boolean instead